lib/cucumber/glue/proto_world.rb

```diff
@@ -38,9 +38,6 @@ def step(name, raw_multiline_arg = nil)
       #     And I am logged in as "Susan"
       #   }
       # @param [String] steps_text The Gherkin snippet to run
-      def steps(steps_text)
```

Comment:
This is a biggie, removing this (I know we said we wanted to)

If we do want to, this should be a breaking change
Also we should probably move the RDoc above

Reply:
I do not understand.
This is part of some kind of public API?
If we are just calling "super", why would there be issues?